lib/formatters/stringExcel.js

```diff
@@ -0,0 +1,7 @@
+const defaulStringFormatter = require('./string');
+
+function stringExcel(opts = { stringFormatter: defaulStringFormatter() }) {
+  return (value) => `"="${opts.stringFormatter(value)}""`;
```

Comment:
A link or comment about why this is the way it is.

Reply:
This is documented in the readme. Do you think that it should also be documented here?

Reply:
Yeah, mainly if there is an issue or someone wants to submit a PR adding features here.